### ISSUE TITLE
Free shipping doesn't work as advertised 

### ISSUE THREAD

**Issue by gldraphael** [GoogleCloudPlatform/microservices-demo#597]
![image](https://user-images.githubusercontent.com/10794213/133376560-85c47f46-c69a-414f-8a42-fd655b43a287.png)

**Comment by NimJay** [comment]
I think this "free shipping" promise is worth fixing.

## BOGO Ads

But somewhat related to this are these other "buy _x_, get (_x_+1)<sup>th</sup> free" false advertisements that show up at the bottom of product pages:
<img width="643" alt="Screen Shot 2021-09-21 at 10 46 08 AM" src="https://user-images.githubusercontent.com/10292865/134192955-192dbd72-4730-4161-8367-dcd73607d9fd.png">

There's 3 such ads in total ([in `AdService.js`](https://github.com/GoogleCloudPlatform/microservices-demo/blob/727987ed391964ffc3ccc70c5c1fcd7e7fc6c2de/src/adservice/src/main/java/hipstershop/AdService.java#L197)).
If I'm not mistake, these ads are supposed to mimic third-party (external) ads — we just happen to link them to Online Boutique products.
So I don't think it's worth fixing. If anything, we could change the wording.

**Comment by NimJay** [comment]
This is a good beginner issue. :)
You can see related code in a slightly similar issue that was fixed, https://github.com/GoogleCloudPlatform/microservices-demo/pull/694/files.
Please ignore my previous comment about the BOGO ads — the BOGO ads aren't worth fixing.

CC: @charlieyu1996

**Comment by NimJay** [comment]
After talking to @charlieyu1996 about this issue, we also thought of deleting the "free shipping" message altogether.
Reasons:
1. Removing code is one of the most productive things we can do as developers. Less maintenance for the future.
1. I don't think any demos or tutorials rely on its existence. I did a quick skim of about 5 Google tutorials (see [Google-internal list](url)) that lists some of these demos/tutorials.
1. The "free shipping" message doesn't really add a lot of value to Online Boutique as a demo app. 

I think this might involve:
* Removing the entire [maroon `<navbar>`](https://github.com/GoogleCloudPlatform/microservices-demo/blob/e1c04f89be61b9af59f6a277539757675b1a60df/src/frontend/templates/header.html#L49) https://github.com/GoogleCloudPlatform/microservices-demo/blob/e1c04f89be61b9af59f6a277539757675b1a60df/src/frontend/templates/header.html#L49
* Removing [relevant CSS](https://github.com/GoogleCloudPlatform/microservices-demo/blob/e1c04f89be61b9af59f6a277539757675b1a60df/src/frontend/static/styles/styles.css#L74).

**Comment by charlieyu1996** [comment]
TLDR: I do agree with Nim here. 

For context:
The issue is in shippingservice/quote.go, we call `CreateQuoteFromCount` , which is the legacy logic that uses the number of items from the cart to calculate the shipping fee. Since we changed the logic, the `count` integer is not useful anymore (We can deprecate it or change it to accept a total cart price). 

Now, the issue is, the function `GetQuote` which calls `CreateQuoteFromCount`. And in `GetQuote` ,we don't have the cart price according to `pb.GetQuoteRequest`. From my understanding, in order to get a product price, we need to traverse through the cart and calculate the price for each product. 

Removing the quote logic would be a sensible approach since it does not add value to the demo.

**Comment by charlieyu1996** [comment]
PR merged
